### ISSUE TITLE
Backend Grid Mollie Subscriptions paging not possible

### ISSUE THREAD

**Issue by ThomasNegeli** [mollie/magento2-subscriptions#60]
Paging should respect the PageSize selected by the customer.
Currently only one page is shown with a maximum of 250 Items and paging is not possible.
This is related to https://github.com/mollie/magento2-subscriptions/issues/59

**Comment by Frank-Magmodules** [comment]
Yes thanks, @ThomasNegeli , this can be considered as one issue I think but will keep it for now like this. Thanks again!

**Comment by Frank-Magmodules** [comment]
Hi @ThomasNegeli , Thanks for opening this issue. This issue wasn’t fully clear to us. Could you check if this still occurs with the latest release, and if so, add more information about what is happening and what you are expecting?
